src/pages/Product/EachProduct.scss

```
@@ -0,0 +1,181 @@
+      font-weight: 400;
+      text-align: center;
+      text-decoration: none;
+      outline: none;
+      color: black;
+
+      div {
+        margin-bottom: 5px;
+      }
+    }
+
+    .productInfo {
+      margin: 20px 0 10px 0;
+      display: flex;
+      justify-content: space-between;
+      width: 400px;
+
+      .mensuration {
+        font-size: 12px;
+
+        .productVolumeKr {
+          font-size: 18px;
+        }
+
+        .productVolumeUs {
+          font-size: 18px;
+        }
+      }
+    }
+
+    .productDescription {
+      margin: 5px 0;
+      padding: 5px 0;
+      width: 400px;
+      min-height: 100px;
+      border-top: 1px solid black;
+      font-size: 14px;
+    }
+
+    .addButton {
+      height: 40px;
+      width: 100%;
+      border: 1px solid black;
+      background-color: white;
+      cursor: pointer;
+    }
+
+    #popup {
+      display: none;
+    }
+
+    #buttonForPopup {
+      display: inline-block;
+      padding: 10px;
+      height: 40px;
+      width: 100%;
+      border: 1px solid black;
+      text-align: center;
+      font-size: 14px;
+      background-color: white;
+      cursor: pointer;
+    }
+
+    .popupWrapperOn {
+      display: block;
+      position: fixed;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      z-index: 100;
+
+      .popupBackground {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        background-color: white;
+        z-index: 2;
+
+        .popupHeader {
+          padding: 12px 35px 12px 19px;
+          color: white;
+          background-color: black;
+        }
+
+        #buttonForClose {
+          position: absolute;
+          top: 0;
+          right: 0;
+          transform: translate(-30%, +35%);
+          width: 28px;
+          height: 28px;
+          border-radius: 50%;
+          font-size: 15px;
+          text-align: center;
+          background-color: black;
+          color: white;
+          z-index: 1;
+          cursor: pointer;
+        }
+
+        .popupContent {
+          width: 600px;
+
+          .popupContentElement {
+            padding: 20px 25px 14px;
+
+            h2 {
+              padding-bottom: 20px;
+              border-bottom: 1px dotted black;
+              font-size: 12px;
+            }
+
+            .popupProductList {
+              margin-top: 20px;
+
+              img {
+                width: 80px;
+                height: 80px;
+              }
+
+              div {
+                display: inline-block;
+                position: relative;
+                top: -70px;
+                margin: 0px 0px 0px 20px;
+                width: 80%;
+
+                span {
+                  font-size: 14px;
+                }
+
+                select {
+                  float: right;
+                }
+              }
+            }
+
+            .totalProducts {
+              .popupOptionDescription {
```

```suggestion
            .popupOptionDescription {
```
스타일 없으면 굳이 써줄 필요 없을거 같아요